.github/workflows/php-code-quality.yml

```diff
@@ -8,9 +8,9 @@ on:
         required: false
         type: "string"
       php:
-        default: 8.0
+        default: "8.0"
         required: false
-        type: number
+        type: string
```

Comment:
YAML treat `8.0` as a floating point number.
So `10.10` will evaluate to `10.1` ❕